frontend/src/metabase/admin/settings/notifications/WebhookForm.tsx

```diff
@@ -0,0 +1,212 @@
+        url: values.url,
+        "auth-method": values["auth-method"],
+        "auth-info": values["auth-info"],
+      },
+    })
+      .unwrap()
+      .then(
+        () => {
+          setFieldError("url", undefined);
+          setTestButtonLabel(t`Success`);
+        },
+        e => {
+          setTestButtonLabel(t`Test failed`);
+          if (typeof e === "string") {
+            setFieldError("url", e);
+          } else if (typeof e.data === "string") {
+            setFieldError("url", e.data);
+          } else if (e.data?.message) {
+            setFieldError("url", e.data.message);
+          }
+        },
+      );
+  };
+
+  return (
+    <FormProvider
+      initialValues={initialValues}
+      onSubmit={onSubmit}
+      validationSchema={validationSchema}
+    >
+      {({ dirty, values, setFieldError }) => (
+        <Form>
+          <Alert
+            variant="light"
+            mb="1.5rem"
+            style={{ backgroundColor: "var(--mb-color-bg-light)" }}
+            px="1.5rem"
+            py="1rem"
+            radius="0.5rem"
+          >
+            <Text>{t`You can send the payload of any Alert to this destination whenever the Alert is triggered. Learn about Alerts`}</Text>
+          </Alert>
+          <Flex align="end" mb="1.5rem" gap="1rem">
+            <FormTextInput
+              name="url"
+              label="Webhook URL"
+              placeholder="http://hooks.example.com/hooks/catch/"
+              styles={styles}
+              style={{ flexGrow: 1 }}
+              maw="21rem"
+            />
+            <Button
+              h="2.5rem"
+              onClick={() => handleTest(values, setFieldError)}
+            >
+              {testButtonLabel}
+            </Button>
+          </Flex>
+          <FormTextInput
+            name="name"
+            label="Give it a name"
+            placeholder="Hooky McHookerson"
+            styles={styles}
+            mb="1.5rem"
+            maw="14.5rem"
+          />
+          <FormTextInput
+            name="description"
+            label="Description"
+            placeholder="Where is this going and what does it send?"
+            styles={styles}
+            mb="1.5rem"
+            maw="21rem"
+          />
+          <FormChipGroup
+            name="auth-method"
+            label="Authentication method"
+            groupProps={{ mb: "1.5rem" }}
+          >
+            <Chip value="none" variant="brand">
+              None
+            </Chip>
+            <Chip value="query-param" variant="brand">
+              Url params
+            </Chip>
+            <Chip value="header" variant="brand">
+              HTTP headers
+            </Chip>
+            <Chip value="request-body" variant="brand">
+              Request body
+            </Chip>
+          </FormChipGroup>
+          {values["auth-method"] !== "none" && (
+            <FormKeyValueMapping
+              name="auth-info"
+              label="Auth info"
```

Comment:
Here too

Reply:
![image](https://github.com/user-attachments/assets/568a493e-ee20-4305-ab64-4a6ba6522e31)

I might be totally off base here but I wasn't sure if the term 'attribute' fit perfectly well in this context. Feels more like this is about key-value pairs or parameters

Reply:
Maybe the heading 'Auth info' is not needed, and the text here could say 'Add a param' (if 'URL params' was selected) or 'Add a header' if Headers was selected? Or is that too annoying to implement?

Reply:
We're implementing a different UI that doesn't include this component anymore, so I'll leave this alone for now